alpha/alpha-server/src/main/resources/schema-postgresql.sql

```diff
@@ -68,3 +68,35 @@ CREATE TABLE IF NOT EXISTS TxTimeout (
 );
 
 CREATE INDEX IF NOT EXISTS saga_timeouts_index ON TxTimeout (surrogateId, expiryTime, globalTxId, localTxId, status);
+
+CREATE TABLE IF NOT EXISTS TccParticipateEvent (
```

Comment:
How about the TCC start event, we need to know it for timeout.

Reply:
I haven't considered well about it.